main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js

```diff
@@ -345,6 +345,7 @@ ReconStandardServicePanel.prototype.start = function() {
         schemaSpace: this._service.schemaSpace,
         type: (type) ? { id: type.id, name: type.name } : null,
         autoMatch: this._elmts.automatchCheck[0].checked,
+        batchSize:this._service.batchSize,
```

Comment:
Technically, the manifest is provided by a third party so the `batchSize` attribute may contain basically anything. It could be useful to add a bit more checks here, only passing it to the backend if it is a positive number.